### ISSUE TITLE
run imdb full locally - P1

### ISSUE THREAD

**Issue by IsabelCanicoNeto** [jmribeiro/dsl-project#20]
configurations  :  3 epochs, full, lr 0.001, han hsan

since 31/12 22h - 

results : 

# ##################### #
# Training hsan #
# ##################### #

Training epoch 1
100%|██████████| 274/274 [5:52:00<00:00, 139.70s/it]
Training loss: 0.6894
Valid acc: 0.5067

Training epoch 2
100%|██████████| 274/274 [6:17:48<00:00, 77.11s/it]
Training loss: 0.6913
Valid acc: 0.4971

Training epoch 3
100%|██████████| 274/274 [6:46:19<00:00, 122.70s/it]
Training loss: 0.6915
Valid acc: 0.4995
Final Test acc: 0.5030
*** Setting up han model on device cpu *** (Done)
*** Setting up adam optimizer *** (Done)

# ##################### #
# Training han #
# ##################### #

Training epoch 1
100%|██████████| 274/274 [7:22:39<00:00, 162.34s/it]
Training loss: 0.5736
Valid acc: 0.8140

Training epoch 2
 11%|█         | 29/274 [1:02:37<6:42:12, 98.50s/it]
